package.json

```diff
@@ -23,9 +23,9 @@
     "clean": "rimraf dist/bundle dist/node",
     "test": "npm run lint && nyc ava test/ && npm run thanks && npm run report-coverage",
     "thanks": "cowsay Hi, thanks for your interest in BigchainDB. We appreciate your contribution!",
-    "release": "./node_modules/release-it/bin/release-it.js --src.tagName='v%s' --github.release --npm.publish --non-interactive",
-    "release-minor": "./node_modules/release-it/bin/release-it.js minor --src.tagName='v%s' --github.release --npm.publish --non-interactive",
-    "release-major": "./node_modules/release-it/bin/release-it.js major --src.tagName='v%s' --github.release --npm.publish --non-interactive",
+    "release": "./node_modules/release-it/bin/release-it.js --git.tagName='v${version}' --github.release --npm.publish --non-interactive",
```

Comment:
I would assume the `release-it` package also needs to be updated so those new commands work. Current version is `12.2.0` https://www.npmjs.com/package/release-it

Reply:
Yes, one of the steps in `RELEASE_PROCESS.md` is:

> We use the [release-it](https://www.npmjs.com/package/release-it) package (from npm) to automate most of the release. Make sure you have a recent version.

Reply:
recent version needs to be defined in the package.json. That's where `release-it` is coming from after doing `npm install`

Reply:
Okay I updated `release-it` to 12.2.0 in `package.json`.

Reply:
👍